### PR TITLE
docs: recommended project structure

### PR DESCRIPTION
**Pull request by yuhan**
### Summary & Motivation

https://dagster-git-yuhan-docs-project-layout-elementl.vercel.app/guides/dagster/recommended-project-structure

### CODE REVIEW

File: docs/content/guides/dagster/recommended-project-structure.mdx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+  - This patten also helps you to easily execute jobs in different environment without code changes. In this example, we dynamically define repository based on the deployment in [`repository.py`](https://github.com/dagster-io/dagster/blob/master/examples/project_fully_featured/project_fully_featured/repository.py) and are able to keep all code the same across testing, local development, staging, and production. Read more about our recommendations in [Transitioning Data Pipelines from Development to Production](/guides/dagster/transitioning-data-pipelines-from-development-to-production)
+
+- TODO: What to do with ops/graphs/jobs???
```
Comment (clairelin135) [inline docs/content/guides/dagster/recommended-project-structure.mdx:36]: I think we decided from the recommended project structure to have an `ops.py` and a `jobs.py`. I can see graphs fitting in with either ops or jobs--we don't necessarily need to address graphs though, we could leave it up to the user. 

Definitely makes sense to me to recommend having all of the ops exist in one file though, for ease of importing and reusing across multiple jobs.

**Review by clairelin135 — COMMENTED** [review]
Added some preliminary comments, but generally this is looking great to me!

**Review by erinkcochran87 — COMMENTED** [review]
This seems to still be in fairly early stages, but I like where it's going. I left some comments on the Vercel preview, but didn't look too closely at grammar for this pass. I think the content itself is looking good, but it could use some structuring to make this easier to parse - I'd love to see it again after that's been added in.

File: docs/content/guides/dagster/recommended-project-structure.mdx (hunk 1 above)
Comment (sryza) [reply docs/content/guides/dagster/recommended-project-structure.mdx:36]: A medium-sized organization could easily have hundreds of ops, which could mean thousands of lines of code, and I think that would be unwieldy to have in one file.

**Comment by yuhan** [comment]
close as https://github.com/dagster-io/dagster/pull/12656 is merged

**Pull request closed without merging**